scripts/process-g6-into-elastic-search.py

```diff
@@ -164,13 +164,31 @@ def attributes(data):
         if "Uptime Institute Tier 4" in data["datacentreTier"]:
             attributes.append({"name": "q32", "q32": "tier4uptimeinstitute"})
 
-    # provising time is 1 Day etc in G6, but
+    # provisioning time is 1 Day etc in G6, but
     # the question is "documented?" in G5
     # so any value provokes true
     if "provisioningTime" in data:
         attributes.append({"name": "q40", "q40": "true"})
 
     #
+    # IaaS
+    #
+
+    if "lot" in data and data["lot"] is "IaaS":
```

Comment:
This really should be `data.get("lot") == "IaaS"`, `is` only works here thanks to short string cache.